dotnet/aspnetcore-runtime-3.1/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                aspnetcore-runtime-3.1
```

Comment:
I think the version is better to indicate without a dot, like this in ports ruby30, python39, etc.

Reply:
That was my thought initially, but according to their release cadence, dotnet 10 is coming in 2025, that is not too far away from today

Reply:
Removing the dot between version components is an old habit in MacPorts that we may never completely get rid of, but many newer ports are not removing the dot because of the confusion that can result when one or both numbers go to a second digit, for example clang-9.0, bazel-3.7, and even some older ports like perl5.28.